docs/data-cookbooks/browse-generation.md

```diff
@@ -0,0 +1,591 @@
+            "duplicate_found": true,
+            "size": 1908635
+          },
+          {
+            "name": "MOD09GQ.A2016358.h13v04.006.2016360104606.hdf.met",
+            "filepath": "MOD09GQ___006/MOD/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf.met",
+            "fileType": "metadata",
+            "bucket": "cumulus-test-sandbox-private",
+            "filename": "s3://cumulus-test-sandbox-private/MOD09GQ___006/MOD/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf.met",
+            "time": 1553027412000,
+            "path": "data",
+            "url_path": "{cmrMetadata.Granule.Collection.ShortName}___{cmrMetadata.Granule.Collection.VersionId}/{substring(file.name, 0, 3)}",
+            "duplicate_found": true,
+            "size": 21708
+          },
+          {
+            "name": "MOD09GQ.A2016358.h13v04.006.2016360104606.jpg",
+            "filepath": "MOD09GQ___006/2017/MOD/MOD09GQ.A2016358.h13v04.006.2016360104606.jpg",
+            "fileType": "browse",
+            "bucket": "cumulus-test-sandbox-protected",
+            "filename": "s3://cumulus-test-sandbox-protected/MOD09GQ___006/2017/MOD/MOD09GQ.A2016358.h13v04.006.2016360104606.jpg",
+            "time": 1553027415000,
+            "path": "data",
+            "url_path": "{cmrMetadata.Granule.Collection.ShortName}___{cmrMetadata.Granule.Collection.VersionId}/{extractYear(cmrMetadata.Granule.Temporal.RangeDateTime.BeginningDateTime)}/{substring(file.name, 0, 3)}",
+            "duplicate_found": true,
+            "size": 1908635
+          },
+          {
+            "name": "MOD09GQ.A2016358.h13v04.006.2016360104606.cmr.xml",
+            "filepath": "MOD09GQ___006/MOD/MOD09GQ.A2016358.h13v04.006.2016360104606.cmr.xml",
+            "fileType": "metadata",
+            "bucket": "cumulus-test-sandbox-protected-2",
+            "filename": "s3://cumulus-test-sandbox-protected-2/MOD09GQ___006/MOD/MOD09GQ.A2016358.h13v04.006.2016360104606.cmr.xml",
+            "url_path": "{cmrMetadata.Granule.Collection.ShortName}___{cmrMetadata.Granule.Collection.VersionId}/{substring(file.name, 0, 3)}"
+          }
+        ],
+        "cmrLink": "https://cmr.uat.earthdata.nasa.gov/search/granules.json?concept_id=G1222231611-CUMULUS",
+        "cmrConceptId": "G1222231611-CUMULUS",
+        "granuleId": "MOD09GQ.A2016358.h13v04.006.2016360104606",
+        "cmrMetadataFormat": "echo10",
+        "dataType": "MOD09GQ",
+        "version": "006",
+        "published": true
+      }
+    ]
+```
+
+You can verify the granules exist within your cumulus instance (search using the Granules interface, check the S3 buckets, etc) and validate that the above CMR entry
+
+
+-----
+
+
+## Build Processing Lambda
+
+This section discusses the construction of a custom processing lambda to replace the contrived example from this entry for a real dataset processing task.
```

Comment:
While implicit, it may be worthwhile to explicitly outline that the output of this lambda is expected to go to `postToCMR`.